bears/vcs/git/GitCommitBear.py

```diff
@@ -291,6 +291,9 @@ def check_issue_reference(self, body,
         if host not in self.SUPPORTED_HOST_KEYWORD_REGEX:
             return
 
+        if not re.search(self.SUPPORTED_HOST_KEYWORD_REGEX[host], body):
+            return
```

Comment:
Sounds good. But what if a user has a commit like this.
```
Shortlog
First line, blablablablablabla
Fix http://gitlab.com/user/repo/issues/1112
Another line, blablablablablabla.
```
And he has the flags `body_close_issue`, `body_close_issue_full_url` and `body_close_issue_on_last_line` enabled? 
Shouldn't the expected output be something that states there's no issue reference in last line? Instead it yields no result.